### DISCUSSION TITLE
2006/03/23/%E7%94%B3%E8%AF%B7%E5%BC%80%E9%80%9A%E4%BA%86%E4%B8%80%E4%B8%AA-google-page

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#79] [Announcements]
# 2006/03/23/%E7%94%B3%E8%AF%B7%E5%BC%80%E9%80%9A%E4%BA%86%E4%B8%80%E4%B8%AA-google-page

http://ruanqizhen.googlepages.com/

https://blog.qizhen.xyz/2006/03/23/%E7%94%B3%E8%AF%B7%E5%BC%80%E9%80%9A%E4%BA%86%E4%B8%80%E4%B8%AA-google-page

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2006年03月29日 03:00 编辑
申请的界面是英文的，不过等到申请下来可以在自己的主页上写英文。但是自己设计页面，肯定比谢blog麻烦一些。

回复
旦阳
2006年03月28日 14:54 编辑
google的主页好用吗？在哪儿申请？
我刚试着从googlepages.com进去，是英文界面的。

回复
